test/schematron/schut-to-xspec-012-out.xspec

```diff
@@ -44,9 +44,9 @@
          label="with text">
          <x:context href="%TEST_BASE%/schematron/schut-to-xspec-012-02.xml"/>
          <x:expect label="assert text: A section inside a div is usually superfluous"
-                   test="exists(Q{http://purl.oclc.org/dsdl/svrl}schematron-output/Q{http://purl.oclc.org/dsdl/svrl}failed-assert[&#xA;            (Q{http://purl.oclc.org/dsdl/svrl}text, Q{http://purl.oclc.org/dsdl/svrl}diagnostic-reference) ! normalize-space(.)&#xA;            = 'A section inside a div is usually superfluous'&#xA;            ])"/>
+            test="exists(Q{http://purl.oclc.org/dsdl/svrl}schematron-output/Q{http://purl.oclc.org/dsdl/svrl}failed-assert[&#xA;            (&#xA;            Q{http://purl.oclc.org/dsdl/svrl}text,&#xA;            Q{http://purl.oclc.org/dsdl/svrl}diagnostic-reference,&#xA;            Q{http://purl.oclc.org/dsdl/svrl}property-reference&#xA;            ) ! normalize-space(.)&#xA;            = 'A section inside a div is usually superfluous'&#xA;            ])"/>
```

Comment:
FYI: This file is used by `test/schematron/schut-to-xspec.xspec`. The changes in this file reflect the changes in `schut-to-xspec.xsl`.